charts/jaeger/README.md

```diff
@@ -68,18 +68,24 @@ data:
 
```

Comment:
at L3, please change http://jaeger.readthedocs.io/en/latest/ to https://www.jaegertacing.io

Reply:
updated 👍